<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ItemConditionDialog</class>
 <widget class="QDialog" name="ItemConditionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>676</width>
    <height>574</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Item condition</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="tabPosition">
      <enum>QTabWidget::South</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="general">
      <attribute name="title">
       <string>General</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item row="0" column="1" rowspan="3">
        <widget class="InventorySlot" name="itemSlot">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>36</width>
           <height>36</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Sunken</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="2" rowspan="3">
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="5">
        <widget class="QStackedWidget" name="stackedWidget">
         <property name="currentIndex">
          <number>1</number>
         </property>
         <widget class="QWidget" name="pre_component">
          <layout class="QGridLayout" name="gridLayout_3">
           <property name="leftMargin">
            <number>7</number>
           </property>
           <property name="topMargin">
            <number>7</number>
           </property>
           <property name="rightMargin">
            <number>7</number>
           </property>
           <property name="bottomMargin">
            <number>7</number>
           </property>
           <item row="1" column="1">
            <widget class="QGroupBox" name="groupBox_2">
             <property name="title">
              <string>Stored enchantments</string>
             </property>
             <layout class="QGridLayout" name="gridLayout_2">
              <property name="leftMargin">
               <number>5</number>
              </property>
              <property name="topMargin">
               <number>5</number>
              </property>
              <property name="rightMargin">
               <number>5</number>
              </property>
              <property name="bottomMargin">
               <number>5</number>
              </property>
              <item row="0" column="0">
               <widget class="ExtendedTableWidget" name="storedEnchantsTable">
                <property name="jsonMode">
                 <enum>ExtendedTableWidget::List</enum>
                </property>
                <property name="columnTitles">
                 <stringlist>
                  <string>Enchantment</string>
                  <string>Levels</string>
                 </stringlist>
                </property>
                <widget class="QFrame" name="storedEnchantsContainer">
                 <layout class="QFormLayout" name="formLayout">
                  <item row="0" column="0">
                   <widget class="QLabel" name="label_5">
                    <property name="text">
                     <string>Enchantment:</string>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="1">
                   <widget class="QComboBox" name="stored_combo"/>
                  </item>
                  <item row="1" column="0">
                   <widget class="QLabel" name="label_6">
                    <property name="text">
                     <string>Levels:</string>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="1">
                   <widget class="NumberProvider" name="stored_levelInput">
                    <property name="sizePolicy">
                     <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                      <horstretch>0</horstretch>
                      <verstretch>0</verstretch>
                     </sizepolicy>
                    </property>
                    <property name="modes">
                     <set>NumberProvider::ExactAndRange</set>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item row="0" column="0" colspan="2">
            <layout class="QFormLayout" name="formLayout_3">
             <item row="1" column="0">
              <widget class="QLabel" name="label_9">
               <property name="text">
                <string>Potion:</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QComboBox" name="potionCombo"/>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label_10">
               <property name="text">
                <string>NBT:</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QLineEdit" name="NBTEdit"/>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Durability:</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="NumberProvider" name="durabilityInput">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>150</width>
                 <height>22</height>
                </size>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <property name="modes">
                <set>NumberProvider::ExactAndRange</set>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="1" column="0">
            <widget class="QGroupBox" name="groupBox">
             <property name="title">
              <string>Enchantments</string>
             </property>
             <layout class="QGridLayout" name="gridLayout">
              <property name="leftMargin">
               <number>5</number>
              </property>
              <property name="topMargin">
               <number>5</number>
              </property>
              <property name="rightMargin">
               <number>5</number>
              </property>
              <property name="bottomMargin">
               <number>5</number>
              </property>
              <item row="0" column="0">
               <widget class="ExtendedTableWidget" name="enchantmentsTable">
                <property name="jsonMode">
                 <enum>ExtendedTableWidget::List</enum>
                </property>
                <property name="columnTitles">
                 <stringlist>
                  <string>Enchantment</string>
                  <string>Levels</string>
                 </stringlist>
                </property>
                <widget class="QFrame" name="enchantsContainer">
                 <layout class="QFormLayout" name="formLayout_2">
                  <item row="0" column="1">
                   <widget class="QComboBox" name="enchant_combo"/>
                  </item>
                  <item row="0" column="0">
                   <widget class="QLabel" name="label_3">
                    <property name="text">
                     <string>Enchantment:</string>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="1">
                   <widget class="NumberProvider" name="enchant_levelInput">
                    <property name="sizePolicy">
                     <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                      <horstretch>0</horstretch>
                      <verstretch>0</verstretch>
                     </sizepolicy>
                    </property>
                    <property name="modes">
                     <set>NumberProvider::ExactAndRange</set>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="0">
                   <widget class="QLabel" name="label_4">
                    <property name="text">
                     <string>Levels:</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="post_component">
          <layout class="QFormLayout" name="formLayout_4">
           <property name="leftMargin">
            <number>7</number>
           </property>
           <property name="topMargin">
            <number>7</number>
           </property>
           <property name="rightMargin">
            <number>7</number>
           </property>
           <property name="bottomMargin">
            <number>7</number>
           </property>
           <item row="0" column="0" colspan="2">
            <widget class="DialogDataButton" name="componentsBtn">
             <property name="text">
              <string>Item components...</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>NBT, potion and enchantments have been moved to sub-predicates from Minecraft 1.20.5 version.</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item row="0" column="0" rowspan="3">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Items / tag:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3" rowspan="3">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Count:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="4" rowspan="3">
        <widget class="NumberProvider" name="countInput">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>22</height>
          </size>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <property name="modes">
          <set>NumberProvider::ExactAndRange</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="predicates">
      <attribute name="title">
       <string>Sub-predicates</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="0" column="0">
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>Not supported yet.</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ExtendedTableWidget</class>
   <extends>QWidget</extends>
   <header>extendedtablewidget.h</header>
   <addpagemethod>setContainer</addpagemethod>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>NumberProvider</class>
   <extends>QFrame</extends>
   <header>numberprovider.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>DialogDataButton</class>
   <extends>QWidget</extends>
   <header>dialogdatabutton.h</header>
  </customwidget>
  <customwidget>
   <class>InventorySlot</class>
   <extends>QFrame</extends>
   <header>inventoryslot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ItemConditionDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>485</x>
     <y>471</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ItemConditionDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>553</x>
     <y>471</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
